src/routes/MainNavigation.jsx

```diff
@@ -21,12 +23,15 @@ const MainNavigation = () => {
                 <Route path='/register'>
                     <RegisterContainer/>
                 </Route>
+                <Route path='/verification'>
+                    <VerificationContainer></VerificationContainer>
+                </Route>
                 <PrivateRoute path={'/analysis'}>
                     <SwitchComponent/>
                 </PrivateRoute>
-                <Route path={'/'}>
+                <PrivateRoute path={'/'}>
```

Comment:
Chcemy mieć na '/' private route? To jest domyślny ekran na laki wchodzi user i pokazywał u się tam ekran logowania